tests/integration/usage_statistics/create_usage_statistics_example_messages.py

```diff
@@ -0,0 +1,18 @@
+from test_usage_statistics_messages import valid_usage_statistics_messages
+import json
+
+if __name__ == "__main__":
+
+    msg = valid_usage_statistics_messages
+
+    # remove top-level items from the messages
+    keys = [k for k in msg.keys()]
+    msg_list = []
+    for key in keys:
+        for item in msg[key]:
+            msg_list.append(item)
+
+    msg_json = json.dumps(msg_list)
+
+    with open("usage_statistics_example_messages.json", "w") as outfile:
+        json.dump(msg_list, outfile)
```

Comment:
If you wanted this to runnable from the command line you would put this in `tasks.py`

```python
import json

import invoke

from tests.integration.test_usage_statistics_messages import valid_usage_statistics_messages

@invoke.task
def gen_usage_stats(ctx, path = "usage_statistics.json"):
    msg = valid_usage_statistics_messages

    # remove top-level items from the messages
    keys = [k for k in msg.keys()]
    msg_list = []
    for key in keys:
        for item in msg[key]:
            msg_list.append(item)

    msg_json = json.dumps(msg_list)

    with open(path, "w") as outfile:
        json.dump(msg_list, outfile)
```

And then this can be run from the command line with `invoke gen-usage-stats`.

Reply:
since gen_usage_stats is a function then should i return the json from the function instead of writing it to the repo?

Reply:
That's up to you.
But yeah I might move the function itself into `/scripts` or maybe a `utils` package.
And then you could import it and call it from the invoke task runner.

Something like...

```python
# invoke.py
import invoke

from scripts import usage_stats

@invoke.task
def gen_usage_stats(ctx, path = "usage_statistics.json"):
   stats = usage_stats()

    with open(path, "w") as outfile:
        json.dump(stats, outfile)
```

The only think `invoke` gives you is the ability to easily rune python code from the command line, and/or run cmd-line tasks from python.

So the above method is runnable from the command line with `invoke gen-usage-stats` or `invoke --path gen-usage-stats my_file.json` 

See https://github.com/great-expectations/great_expectations/pull/5506 for more examples (btw this should be merging today 🤞 )

Reply:
bombing this pr, this branch, and my poor venv that i could not get working correctly based on the requirements files.